### DISCUSSION TITLE
Notifier filetype

### DISCUSSION THREAD

**Discussion by dpetka2001** [folke/snacks.nvim#6] [Q&A] [answered]
In LazyVim because the notifier window is set to `filetype = "markdown"`, `mini.indentscope` shows indents when you go with your cursor into the notification window (by doing `<c-w>w` for example). I tried the following
```lua
  {
    "folke/snacks.nvim",
    opts = {
      styles = {
        notification = {
          bo = { filetype = "snacks_notify" },
        },
      },
    },
  },
```
and the filetype changes to `snacks_notify` and added to the local LazyVim forked repo in `mini.indentscope` plugin spec the pattern `snacks_notify`, but it still shows indents. While my cursor is in the notification window if I do `:lua miniindentscope_disable = true` it correctly disables the indents. 

Maybe the `FileType` autocmd does not catch the filetype of the notification window? Is there a workaround other than adding `BufEnter` to the autocmd?

**Comment by dpetka2001** [comment]
It seems that FileType event does not trigger when the notification window is rendered. If I do `:doautocmd FileType` then the indents disappear without me having to manually `:lua miniindentscope_disable = true` (although I still have to move the cursor to trigger a redraw update), since I manually trigger the event.

**Reply by folke** [reply, marked as answer]
Should be fixed now. Notifs will have `ft=snacks_notif`.

Update LazyVim & snacks for the changes :)

**Reply by dpetka2001** [reply]
Yep, all good. Thank you!! :smile:

**Comment by folke** [comment]
Right, that;s a bug. The `filetype` probably shouldn't be set, since snacks.win does the rendering anyway.

I use `noautocmd` to open the window,, so that's why the autocmds dont trigger
